### ISSUE TITLE
UI Redesign - Mobile - My Files

### ISSUE THREAD

**Issue by ksanty** [atsign-foundation/atmosphere_pro#218]
**Description**
The task is to implement the new design for the Files menu in the mobile app. 

**Assets and Design Screens**
All assets and design screens have already been created.

- The Figma design can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=2%3A3
- The Assets required can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=0%3A1

For help with using Figma please watch this short tutorial video
https://user-images.githubusercontent.com/69275762/192322842-2ac40f88-c396-4a66-933b-3c8abae516c3.mov

**Additional context**
This work should be done on the UI_Redesign branch https://github.com/atsign-foundation/atmosphere_pro/tree/ui_redesign

**Comment by ruqaiyasattar** [comment]
interested

**Comment by ksanty** [comment]
> interested

Great!  Let us know if you have any questions.

**Comment by Ayush783** [comment]
Hey. Can I also work on this or is there a ticket reservation?

**Comment by AM1CODES** [comment]
I would love to contribute to this. Is there a way we can work on this together or we can all give this a shot and make a PR?

**Comment by tinashe404** [comment]
@Ayush783 yes you can work on it if you'd like. I've assigned it to you too

**Comment by tinashe404** [comment]
@AM1CODES that is allowed, however we wouldn't be able to make the connection for you. If you reach out to the other contributors and they are willing to work with you, then please go ahead.

**Comment by Ayush783** [comment]
Great! Thanks

**Comment by AM1CODES** [comment]
Thank you!

**Comment by Dariamarg** [comment]
Hey @sonle-geekyants! I completely missed your message:

1. I adjusted (simplified) the design to not include icons 
<img width="331" alt="Screenshot 2023-01-18 at 12 30 02 AM" src="https://user-images.githubusercontent.com/84993403/213034924-606e1acc-f6eb-42f7-b978-57b8e5f07e22.png">

2. [Here](https://www.figma.com/file/eKGFZ2t0hi03LT07n1Fx36/AtmospherePro-V2?node-id=2%3A2&t=Z7rSHlPSQTMHvtis-1) is the Figma File
<img width="239" alt="Screenshot 2023-01-18 at 12 30 46 AM" src="https://user-images.githubusercontent.com/84993403/213035010-f6868666-01d1-4232-bd07-941a22549a60.png">

3. On the same Figma file as above
<img width="338" alt="Screenshot 2023-01-18 at 12 31 40 AM" src="https://user-images.githubusercontent.com/84993403/213035111-3c688d2c-ba4a-4bbc-8a55-8e288eff3bd2.png">

4. Search Icon (same Figma):
<img width="520" alt="Screenshot 2023-01-18 at 12 35 53 AM" src="https://user-images.githubusercontent.com/84993403/213035688-0ea822a9-5381-4017-b002-f3c79b5b76de.png">

5. Chat flow (same Figma):
- Icon only appears if there is a message associated
- Second screen is how it would look like
<img width="709" alt="Screenshot 2023-01-18 at 12 38 47 AM" src="https://user-images.githubusercontent.com/84993403/213036024-d7f58f9f-31b8-4ea6-9f8e-04462a2c9364.png">
